actions/utils.js

```diff
@@ -121,12 +123,46 @@ function getDocPathFromUrl(url) {
     return `${path}.docx`;
 }
 
+async function updateStatus(projectExcelPath, status) {
+  const logger = getAioLogger();
+  // md5 hash of the config file
+  try {
+    const hash = crypto.createHash('md5').update(projectExcelPath).digest('hex');
+    logger.info(`Adding status to aio state lib with hash -- ${hash} - ${status}`);
+    const state = await stateLib.init();
+    state.put(hash, status);
```

Comment:
From what I understand, looks like this is just storing the status string. We need to store more data in the same project than just the action status. So the value should be more like a JSON object so that we can append additional data in it. And also the TTL needs to be set as the data is only stored for 24 hours by default. Perhaps 3 month TTL?

Here is an example in their [doc](https://github.com/adobe/aio-lib-state)
```
await state.put('key', { anObject: 'value' }, { ttl: -1 })
```

So in our case, it should be something like:
```
{
  "copyAction": {
    "status": "in-progress",
    "activationId": "iwuaeyqiwejdla"
  },
  "promoteAction": {
    "status": "not-started"  
  },  
  "somethingElse": "data"
}
```